<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>XINDA SERVO SYSTEM</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>1241</width>
      <height>661</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_Home">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QSlider" name="horizontalSlider_gauge">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>331</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>580</y>
        <width>691</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_Silder">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>620</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="AnalogGaugeWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QLabel" name="label_Gauge">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_File">
     <attribute name="title">
      <string>File</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_Paramter">
     <attribute name="title">
      <string>Paramter</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_EDIT">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>251</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);</string>
    </property>
    <property name="text">
     <string>Home Page</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EDIT_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>251</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);</string>
    </property>
    <property name="text">
     <string>File Manage</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EDIT_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>251</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EDIT_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>251</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);</string>
    </property>
    <property name="text">
     <string>Paramter</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalogGaugeWidget</class>
   <extends>QWidget</extends>
   <header location="global">analoggaugewidget</header>
   <container>1</container>
   <slots>
    <signal>valueChanged(int)</signal>
    <slot>update_angle_offset(int)</slot>
    <slot>update_value(int)</slot>
    <slot>update_value2(int)</slot>
    <slot>set_start_scale_angle(int)</slot>
    <slot>set_total_scale_angle_size(int)</slot>
    <slot>set_MinValue(int)</slot>
    <slot>set_MaxValue(int)</slot>
    <slot>set_NeedleColor(int)</slot>
    <slot>set_scala_main_count(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>fOpenFile()</slot>
  <slot>fSaveFile()</slot>
  <slot>fLanguageChange()</slot>
  <slot>fMainTableChange()</slot>
  <slot>fMainTableClick()</slot>
  <slot>f4RowClick()</slot>
  <slot>fEDIT_ButtonClick()</slot>
  <slot>fAdvancedParam_ButtonClick()</slot>
  <slot>fWORK_ButtonClick()</slot>
  <slot>fTEST_ButtonClick()</slot>
  <slot>fSTEP_ButtonClick()</slot>
  <slot>fMODY_ButtonClick()</slot>
  <slot>fHELP_ButtonClick()</slot>
  <slot>fFile_OnClik()</slot>
  <slot>f5RowClick()</slot>
  <slot>fDownload_FileFlash()</slot>
 </slots>
</ui>
